### DISCUSSION TITLE
v-autocomplete issue

### DISCUSSION THREAD

**Discussion by piyush-angelone** [vuetifyjs/vuetify#16294] [General]
@update: search event call on focus and blur event
 is this expected behavior or bug?

**Comment by BilalShah3** [comment]
i also have the same problem did you find any solution

**Reply by yuwu9145** [reply]
- It emits on focus because it has to show the input that has selected value by default (e.g. in this [demo](https://play.vuetifyjs.com/#eNpVkMFqwzAQRH9l0UWXxKa0hTaogRBof6DQQ5KDaq+DQNKKlWxaQv69stxg5zaaeSt253ARkZt6F0I19Cg2QiV0weqE26MHUMNah1BkeTTkkzYe+d+aiD5RQ3kKE95sAKu/0b4dxW6Zijl31KJdD9r2mKm9tqYj9kYvmY3J28QcH+QMyBXIPVli3dKo37M0bbE/kPg8EZ/4o+Movn7JGX+Wp/njrarvt74dmO37CwtYClD1ohgVGzYhQcTUT+0YF4gTXICxgyt0TA5kblSOkxOdOXFdicfqqXp4FaN4rl7E6Q/cIHpt), "Clifornia" must be show up in input when focus)
- It emits on blur because it has to reset search when using chips/selection slot, as in this case, default selected value is displayed in chips, input must display as empty next to it [demo](https://play.vuetifyjs.com/#eNpVkcFqwzAMhl9F+JJLmjC2wVayQClsLzDYoe3BTZTW4FjGVkJL6bvPThqSXsyvX58t9Ht3E95V+cbarO9QrEXB2FotGcu9ASj6lbR2kENRkWGpDLqHFUxb8tUieGoRqs4ztcB4YTDxYAI+B/+sbArS1HDUnSvy6cVxQMdUURiKjJMNoOUR9ddebJZdMfdbqlGveqk7DNRWatWQM0oumTjWz+Vahd18oHfJzCcpJFvS5GRNUX8HqerB/kFyp5H4xYv0UfxdqVXmlBzmOWWRPy8xxRXs57wGcIizyBcxF75yyjJ45G5MRrWWHMMNHDZwh8aFUJPwP0m8OdKBE/dUvGZv2cuniOI9+xCHf/tjk2w=)


Both focus and blur have reason to have current behaviour


Apart from the events feeling somewhat redundant, what other issues does it cause?
